dist/js/stacks.js

```diff
@@ -4570,11 +4570,12 @@ var Stacks;
         TooltipController.prototype.scheduleShow = function (dispatcher) {
             var _this = this;
             if (dispatcher === void 0) { dispatcher = null; }
-            this.activeTimeout = setTimeout(function () { return _this.show(dispatcher); }, 300);
+            window.clearTimeout(this.activeTimeout);
```

Comment:
This code is "all new". Looks like the tooltip PR got merged with without the latest version of the compiled code. Not an issue since the added code is "optional", but it's something we'll need to make sure to check in the future.